### DISCUSSION TITLE
How to interact with a tooltip?

### DISCUSSION THREAD

**Discussion by aborruso** [neocarto/bertin#115] [Q&A] [answered]
Hi,
I'm building [my firs Bertin map](https://observablehq.com/d/fa939367237d9243) and it's a fantastic tool and very well documented. A thousand thanks!

I have enabled in this map a tooltip: currently it's the name of the cities, using the field `$citta`.

![image](https://github.com/neocarto/bertin/assets/30607/533e1819-5638-4ec3-aac6-b4845c8932b3)

I would like to add a text that contains a hyperlink in which the user can click to open a related PDF (it is a map of heat wave warnings).

Then I have to question:

- how to insert a text that has some hyperlink (I have it in the field `URL`, of the `citta` layer)?
- how to stick a tooltip? Because if I insert a text with some hyperlink, I have to allow the user to click on it, without the tooltip disappearing as soon as he moves the mouse.

Thank you

**Comment by neocarto** [comment, marked as answer]
Well, for the moment it's not possible. I'll think about it. Can you post an issue?

**Reply by aborruso** [reply]
> Well, for the moment it's not possible. I'll think about it. Can you post an issue?

the issue https://github.com/neocarto/bertin/issues/117
